src/AccessibilityInsights/MainWindow.xaml

```diff
@@ -249,7 +249,7 @@
                             <Button Name="btnMax" Width="35" Height="28" BorderBrush="{x:Null}"  HorizontalContentAlignment="Center" Click="btnMax_Click"
                                     AutomationProperties.Name="{x:Static properties:Resources.btnMaxAutomationPropertiesName}" Style="{StaticResource btnMinMax}">
                                 <Grid Width="35" Height="28" Background="{Binding Background, ElementName=btnMax}">
-                                    <fabric:FabricIconControl GlyphName="Checkbox" GlyphSize="Small" HorizontalAlignment="Center" VerticalAlignment="Center" Foreground="{DynamicResource ResourceKey=IconBrush}"/>
+                                    <fabric:FabricIconControl Name="btnMaxFabric" GlyphName="Checkbox" GlyphSize="Small" HorizontalAlignment="Center" VerticalAlignment="Center" Foreground="{DynamicResource ResourceKey=IconBrush}"/>
```

Comment:
>GlyphSize [](start = 103, length = 9)

A user can force the app to launch maximized (see [https://www.computerhope.com/tips/tip201.htm](https://www.computerhope.com/tips/tip201.htm)). Have we confirmed that the correct glyph displays if this occurs?

Reply:
fixed